webpack.universal.js

```diff
@@ -84,26 +86,12 @@ const webpackconfig = {
   },
   plugins: [
     new webpack.optimize.DedupePlugin(),
-    new webpack.optimize.UglifyJsPlugin({
-      beautify: false,
-      comments: true,
-      compress: {
-        warnings: false,
-        unused: true,
-        dead_code: true,
-        drop_console: true,
-        comparisons: true,
-        conditionals: true,
-      },
-
-      mangle: {
-        screw_ie8: true,
-        keep_fnames: true,
-      },
-    }),
     new ExtractTextPlugin('../common.css', {
       allChunks: true,
     }),
+    new webpack.DefinePlugin({ 'process.env': {
+      IMAGE_STORAGE_URL: JSON.stringify(process.env.IMAGE_STORAGE_URL),
```

Comment:
Seems weird to pass as a var to webpack. In fact is weird. We should just a config while in the app itself.

But works for now.